docs/ECLWatch/TheECLWatchMan.xml

```diff
@@ -1133,150 +1130,169 @@
+                DfuXrefAccess access to FULL for only those
+                users.</para><para> </para></entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>
 
-      <itemizedlist>
-        <listitem>
-          <para>Press the <emphasis role="bold">Generate</emphasis>
-          button.</para>
+        <para>To generate a list:</para>
 
-          <informaltable colsep="1" frame="all" rowsep="1">
-            <?dbfo keep-together="always"?>
+        <itemizedlist>
+          <listitem>
+            <para>Press the <emphasis role="bold">Generate</emphasis>
+            button.</para>
 
-            <tgroup cols="2">
-              <colspec colwidth="49.50pt" />
+            <informaltable colsep="1" frame="all" rowsep="1">
+              <?dbfo keep-together="always"?>
 
-              <colspec />
+              <tgroup cols="2">
+                <colspec colwidth="49.50pt" />
 
-              <tbody>
-                <row>
-                  <entry><inlinegraphic fileref="images/tip.jpg" /></entry>
+                <colspec />
 
-                  <entry><para>Sasha Server typically runs Xref at the times
-                  scheduled when deployed.</para><para> </para></entry>
-                </row>
-              </tbody>
-            </tgroup>
-          </informaltable>
-        </listitem>
-      </itemizedlist>
+                <tbody>
+                  <row>
+                    <entry><inlinegraphic fileref="images/tip.jpg" /></entry>
 
-      <para>To view results:</para>
+                    <entry><para>Sasha Server typically runs Xref at the times
+                    scheduled when deployed.</para><para> </para></entry>
+                  </row>
+                </tbody>
+              </tgroup>
+            </informaltable>
+          </listitem>
+        </itemizedlist>
 
-      <itemizedlist>
-        <listitem>
-          <para>Click on one of the hyperlinks (Found Files, Orphan Files,
-          Lost Files, Directories, or Errors/Warnings).</para>
-        </listitem>
-      </itemizedlist>
+        <para>To view results:</para>
 
-      <sect2 id="ECLWatch_WorkingWithXref">
-        <title>Working with XREF results:</title>
+        <itemizedlist>
+          <listitem>
+            <para>Click on one of the tabs (Found Files, Orphan Files, Lost
+            Files, Directories, or Errors/Warnings).</para>
+          </listitem>
+        </itemizedlist>
 
-        <para>After XRef completes you will see a list of available reports.
-        Click on one of the links to see the results page of each type.</para>
+        <sect3 id="ECLWatchXREFMultiThor">
+          <title>XREF with multiple Thor clusters</title>
 
-        <figure>
-          <title>XRef Errors</title>
+          <para>XREF runs on the primary Thor cluster. For a configuration
+          with more than one Thor cluster, ECL Watch only displays the Thor
```

Comment:
>For a configuration with more than one Thor cluster

Need to be clear that it's not any configuration with greater than 1 Thor, but clusters with greater than 1 Thor on the same physical group.

Perhaps:

> For a configuration with more than one Thor cluster on the same physical node group, ...